app/jinja_filters.py

```diff
@@ -66,6 +70,12 @@ def format_multilined_string(context, value):
     return result
 
 
+@blueprint.app_template_filter()
+def get_current_date(date_format='%-d %B %Y'):
```

Comment:
As discussed - could you also pull these 3 instances of this date format out into a variable?